src/sfizz/CCMap.h

```diff
@@ -42,7 +43,7 @@ class CCMap {
      */
     const ValueType& getWithDefault(int index) const noexcept
     {
-        auto it = container.find(index);
+        auto it = absl::c_find_if(container, [&](auto&& pair){ return pair.first == index; });
```

Comment:
If this is ordered, it could be binary search for `O(log N)`. cf. `std::lower_bound`

Reply:
We could add sorting for sure.

Reply:
>I could add sorting for sure.

I meant also on lookup, such that the search becomes O(log N)

Reply:
Ho right, so in the end this ends up having the same lookup performance as a map?

Reply:
It's possibly better than `map` yes, at lookup, while worse at inserting.